src/molecules/content/ComparatorSingleModal.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { registeredTextStyle, UrbiFontStyles } from 'src/utils/textStyles';
+import { layoutStyles } from 'src/utils/styles';
+
+const styles = StyleSheet.create({
+  Wrapper: {
+    height: 72,
+    ...layoutStyles.ColumnJustifyStart,
+    alignItems: 'flex-start',
+    width: '60%',
+  },
+  Title: registeredTextStyle('title' as keyof UrbiFontStyles),
```

Comment:
these `as keyof` casts shouldn't be needed. Do you get compiler errors?

Reply:
Thanks. Commit 1bef5d1